jerry-core/api/jerry.c

```diff
@@ -323,7 +323,6 @@ jerry_parse (const jerry_char_t *source_p, /**< script source */
   if (JERRY_CONTEXT (jerry_init_flags) & JERRY_INIT_MEM_STATS_SEPARATE)
   {
     jmem_stats_print ();
-    jmem_stats_reset_peak ();
```

Comment:
no preference. I was just interested what the reason was. this deprecation could even go into a separate PR, but I'd like to learn what compatibility we'd like to keep?